src/main.tf

```diff
@@ -1,8 +1,5 @@
 locals {
-  connect_mode = true ? "PRIVATE_SERVICE_ACCESS" : "DIRECT_PEERING"
-
-  tier                = var.tier == "Single Instance" ? "BASIC" : "STANDARD_HA"
-  is_highly_available = local.tier == "STANDARD_HA"
+  is_highly_available = var.tier == "STANDARD_HA"
```

Comment:
Was this tested locally? I see GH action failure: `Reference to undeclared input variable`

Reply:
Alright, so this worked locally because I had `read_replicas` in my local `_params.auto`. I reworked the `massdriver.yaml` to nest `read_replicas` under `availability`, which is guranteed to exist. SO.... we can now test against `var.availability.tier` and `var.availability.read_repliacs` won't throw an error unless we try to use the value. It's how a lot of our "enable XYZ" stuff works too.